jupyter-js-widgets/src/widget_int.js

```diff
@@ -200,23 +203,43 @@ var IntSliderView = widget.DOMWidgetView.extend({
             var readout = this.model.get('readout');
             if (readout) {
                 this.readout.style.display = '';
+                window.setTimeout(function() {
+                    if (that.readout_overflow()) {
+                        that.readout.classList.add('overflow');
+                    } else {
+                        that.readout.classList.remove('overflow');
+                    }
+                });
             } else {
                 this.readout.style.display = 'none';
             }
+
         }
         return IntSliderView.__super__.update.apply(this);
     },
 
     /**
+     * Readout purpling
```

Comment:
I don't think non-Bloomberg users are going to understand the "purpling" reference in the comment.